scss/detail.scss

```diff
@@ -1,74 +1,163 @@
-.container-book {
-  background-color: $white-secondary;
-  box-shadow: 4px 4px 10px 0 $shadow;
-  display: flex;
-  height: 416px;
-  width: 900px;
-}
-
-.container-img {
-  position: relative;
-  height: 100%;
-  max-width: 309px;
-  padding: 24px;
-  width: 100%;
+// variables
+$badge-height: 119px;
+$badge-width: 95px;
+$container-height: 416px;
+$container-width: 900px;
 
-  .badge {
-    display: flex;
-    height: 119px;
-    width: 95px;
-    position: absolute;
-    transform: rotate(15deg);
-    right: -6px;
-    top: 0;
-    padding: 11px;
+.back-arrow {
+  display: flex;
+  align-items: center;
+  width: 90%;
+  height: 20px;
+  margin-bottom: 33px;
+  cursor: pointer;
+  
+  h4::before {
+    content: url(./assets/back-arrow-20.png);
+    height: 20px;
+    width: 20px;
+    z-index: 100;
+    margin-right: 12px;
   }
 }
 
-.container-detail {
-  height: auto;
-  padding: 24px;
-  width: auto;
-}
-
-.header-detail {
-  align-items: center;
-  border-bottom: 4px solid $green-lemon;
+.container-book {
+  background-color: $white-secondary;
+  box-shadow: 4px 4px 10px 0 $shadow;
   display: flex;
-  padding-bottom: 12px;
-  width: 100%;
-}
+  height: $container-height;
+  width: $container-width;
 
-.title-book {
-  color: $black-default;
-  font-weight: 700;
-  margin-left: 8px;
-  font-size: 32px;
-  line-height: 32px;
-}
+  .container-img {
+    position: relative;
+    height: 416px;
+    max-width: 309px;
+    padding: 24px;
+    width: 309px;
+  
+    .badge {
+      display: flex;
+      height: $badge-height;
+      width: $badge-width;
+      position: absolute;
+      transform: rotate(15deg);
+      padding: 10px;
+      right: -5px;
+      top: 0px;
+    }
+  }
 
-.title-gender {
-  color: $grey;
-  font-weight: 700;
-  padding-left: 8px;
-  line-height: 28px;
-  font-size: 28px;
+  .container-detail {
+    height: auto;
+    padding: 24px;
+    width: auto;
```

Comment:
I'm sure you can remove the `height` and `width` rules since `auto` is the default value for both